src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,85 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+
+    private Node<K, V>[] nodeArray = new Node[DEFAULT_CAPACITY];
+    private int size = 0;
 
     @Override
     public void put(K key, V value) {
-
+        resize();
+        Node<K, V> noda = nodeArray[getIndex(key)];
+        if (noda == null) {
+            Node<K, V> currentNode = new Node<>(key, value, null);
+            nodeArray[getIndex(key)] = currentNode;
+            size++;
+        } else {
```

Comment:
```suggestion
            size++;
            return;
       }
```
nested code is hard to read, [return early](https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement)